Ts/core 1620/unify reupload path

oleiman opened the pull request:
<!--
See https://github.com/redpanda-data/redpanda/blob/dev/CONTRIBUTING.md#pull-request-body
for more details and examples of what is expected in a PR body.

Content in this top section is REQUIRED. Describe, in plain language, the motivation
behind the change (bug fix, feature, improvement) in this PR and how the included
commits address it.

Add the GitHub keyword `Fixes` to link to bug(s) this PR will fix, e.g.
  Fixes #ISSUE-NUMBER, Fixes #ISSUE-NUMBER, ...

If this PR is a backport, link to the original with `Backport of PR`, e.g.
  Backport of PR #PR-NUMBER
-->
built off https://github.com/redpanda-data/redpanda/pull/25731
## Backports Required

<!-- Checking at least one of the checkboxes is REQUIRED if this PR is not a backport. -->

- [x] none - not a bug fix
- [ ] none - this is a backport
- [ ] none - issue does not exist in previous branches
- [ ] none - papercut/not impactful enough to backport
- [ ] v25.1.x
- [ ] v24.3.x
- [ ] v24.2.x
- [ ] v24.1.x

## Release Notes

* none
* 
<!--
If the changes in this PR do not need to be mentioned in the release
notes, then don't add a sub-section and simply list `none`, e.g.

* none

Otherwise, adding a sub-section or `none` is REQUIRED if the PR is not a backport PR.
If this is a backport PR, adding contents to this section will override
the release notes section inherited from the original PR to dev.

Add one or more of the sub-sections with a short description bullet
point of the change, e.g.

### Bug Fixes

* Short description of the bug fix if this is a PR to `dev` branch.

### Features

* Short description of the feature. Explain how to configure.

### Improvements

* Short description of how this PR improves existing behavior.

-->

Lazin committed:
archival: Add method that extracts stream

from the 'segment_collector' intance

Signed-off-by: Evgeny Lazin <4lazin@gmail.com>

Lazin committed:
archival: Add test that checks

...make_puload_candidate_stream method

Signed-off-by: Evgeny Lazin <4lazin@gmail.com>

Lazin committed:
archival: Add offset skipping

"make_upload_candidate_stream" can skip offset now. The behavior is
similar to how ntp_archiver works in this case.

Signed-off-by: Evgeny Lazin <4lazin@gmail.com>

Lazin committed:
archival: Add test case for offset skipping

Signed-off-by: Evgeny Lazin <4lazin@gmail.com>

Lazin committed:
archival: New segment_collector mode

Adds new collector mode which can be used to upload new segments. This
makes 'segment_collector' suitable to replace some logic inside the
'archival_policy'. The long term goal here is to make 'segment_reupload'
the only code path which is used to find upload candidates. Then the
'ntp_archiver' should start using method of the segment collector which
returns a stream instead of the set of segments. When its done the
internals of the 'segment_collector' could be updated to use
'log_reader' instead of iterating through the segments.

Signed-off-by: Evgeny Lazin <4lazin@gmail.com>

Lazin committed:
archival: Swap segment collector implementation

Previously, the search of segments that can be uploaded to the cloud
storage was implemented inside the archival_policy. For non-compacted
segments the archival_policy used its internal implementation but for
compacted reuploads it was using segment_collector.

This commit introduces new segment serach logic that uses
segment_collector so both compacted and non-compacted uploads are going
through the collector now.

Signed-off-by: Evgeny Lazin <4lazin@gmail.com>

Lazin committed:
archival: Remove dead code

Signed-off-by: Evgeny Lazin <4lazin@gmail.com>

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64597-->
<!--4718d2de-c386-4351-a101-feac29c569cb-->
<!--8dc3a1af-ec42-4357-a7b8-c37fc0a8d6da-->
<!--title-->
## Retry command for [Build#64597](https://buildkite.com/redpanda/redpanda/builds/64597)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```

/ci-repeat 1
tests/rptest/tests/shadow_indexing_compacted_topic_test.py::TSWithAlreadyCompactedTopic.test_initial_upload
```
<!--/body-->

vbotbuildovich commented:
<!--TEST RESULTS-->
<!--468c540f-3faf-41cb-b5b7-6294096d82c8-->
<!--TEST RESULTS-->
<!--bee293f5-fedc-4a3e-a0fc-e73bc1df040d-->
<!--TEST RESULTS-->
<!--62e4e789-3ed8-4eac-b3dc-2257b1095811-->
<!--TEST RESULTS-->
<!--b144d806-7aa3-4819-8114-787c0ed33600-->
<!--TEST RESULTS-->
<!--633974e5-802b-41d3-b8fc-666b633f3bdb-->
<!--TEST RESULTS-->
<!--6f4eb107-00bc-46a0-8017-7449769ab1fc-->
<!--TEST RESULTS-->
<!--de84fb69-78c5-4d90-85b1-02ba0f45a5b4-->
<!--TEST RESULTS-->
<!--d7d5fee1-96b8-4cf1-81a0-c36737639feb-->
<!--TEST RESULTS-->
<!--4f2f5fa0-543e-4c00-b1d4-8e253f030e4c-->
<!--title-->
## CI test results
<!--/title-->
<!--body-->
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64597">build#64597</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_creating_and_listing_migrations | ducktape | https://buildkite.com/redpanda/redpanda/builds/64597#01963d5a-1e57-4117-a28d-cd2e6833ef04 | FLAKY | 1/2 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_higher_level_migration_api | ducktape | https://buildkite.com/redpanda/redpanda/builds/64597#01963d3f-beca-44bb-9117-e6e702a146aa | FLAKY | 1/4 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_higher_level_migration_api | ducktape | https://buildkite.com/redpanda/redpanda/builds/64597#01963d5a-1e59-4f8a-a64f-b279c7d6a2c0 | FLAKY | 1/2 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64597#01963d3f-becb-4072-a529-5b4dd3d52711 | FLAKY | 1/2 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.True | ducktape | https://buildkite.com/redpanda/redpanda/builds/64597#01963d3f-bec9-4440-a771-91c8eb110a8f | FLAKY | 1/2 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_aborted_tx.recovery_overrides=.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64597#01963d5a-1e59-4f8a-a64f-b279c7d6a2c0 | FLAKY | 1/2 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_aborted_tx.recovery_overrides=.retention.local.target.bytes.1024.redpanda.remote.write.True.redpanda.remote.read.True.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64597#01963d5a-1e56-439b-8ca3-71f2a86d0ff2 | FLAKY | 1/9 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_aborted_tx.recovery_overrides=.retention.local.target.bytes.1024.redpanda.remote.write.True.redpanda.remote.read.True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64597#01963d5a-1e57-4117-a28d-cd2e6833ef04 | FLAKY | 1/2 |
| rptest.tests.scaling_up_test.ScalingUpTest.test_scaling_up_with_recovered_topic | ducktape | https://buildkite.com/redpanda/redpanda/builds/64597#01963d5a-1e57-4117-a28d-cd2e6833ef04 | FLAKY | 1/2 |
| rptest.tests.shadow_indexing_compacted_topic_test.TSWithAlreadyCompactedTopic.test_initial_upload | ducktape | https://buildkite.com/redpanda/redpanda/builds/64597#01963d3f-beca-44bb-9117-e6e702a146aa | FAIL | 0/20 |
| rptest.tests.shadow_indexing_compacted_topic_test.TSWithAlreadyCompactedTopic.test_initial_upload | ducktape | https://buildkite.com/redpanda/redpanda/builds/64597#01963d5a-1e59-4f8a-a64f-b279c7d6a2c0 | FAIL | 0/20 |
| rptest.tests.upgrade_test.UpgradeBackToBackTest.test_upgrade_with_all_workloads.single_upgrade=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64597#01963d3f-becb-4072-a529-5b4dd3d52711 | FLAKY | 1/3 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64707">build#64707</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.cloud_storage_timing_stress_test.CloudStorageTimingStressTest.test_cloud_storage_with_partition_moves.cleanup_policy=delete | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#01964600-f35c-410f-bb6d-915c68f22fa7 | FLAKY | 19/21 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_higher_level_migration_api | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#01964600-f35e-46bd-a2d3-3fe8d765e0f7 | FLAKY | 11/21 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.True | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#019645fc-4c13-4dfe-abf4-48e13517dc4b | FLAKY | 14/21 |
| rptest.tests.datalake.compaction_test.CompactionTest.test_compaction.cloud_storage_type=CloudStorageType.S3.catalog_type=CatalogType.NESSIE | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#01964600-f35e-46bd-a2d3-3fe8d765e0f7 | FLAKY | 12/21 |
| rptest.tests.datalake.compaction_test.CompactionTest.test_compaction.cloud_storage_type=CloudStorageType.S3.catalog_type=CatalogType.REST_HADOOP | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#01964600-f35b-4852-b807-fabe7a160990 | FLAKY | 13/21 |
| rptest.tests.datalake.throttling_test.DatalakeThrottlingTest.test_basic_throttling.cloud_storage_type=CloudStorageType.S3.catalog_type=CatalogType.NESSIE | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#01964600-f35d-44e8-aad2-585383bf51a2 | FLAKY | 17/21 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_aborted_tx.recovery_overrides=.retention.local.target.bytes.1024.redpanda.remote.write.True.redpanda.remote.read.True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#01964600-f35c-410f-bb6d-915c68f22fa7 | FLAKY | 10/21 |
| rptest.tests.offset_for_leader_epoch_archival_test.OffsetForLeaderEpochArchivalTest.test_querying_archive | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#019645fc-4c13-4dfe-abf4-48e13517dc4b | FAIL | 0/21 |
| rptest.tests.offset_for_leader_epoch_archival_test.OffsetForLeaderEpochArchivalTest.test_querying_archive | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#01964600-f35d-44e8-aad2-585383bf51a2 | FAIL | 0/21 |
| rptest.tests.read_replica_e2e_test.ReadReplicasUpgradeTest.test_upgrades.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#019645fc-4c16-43f9-b7f0-aa2e4c2df7a5 | FAIL | 0/21 |
| rptest.tests.read_replica_e2e_test.ReadReplicasUpgradeTest.test_upgrades.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#01964600-f35e-46bd-a2d3-3fe8d765e0f7 | FAIL | 0/21 |
| rptest.tests.shadow_indexing_compacted_topic_test.ShadowIndexingCompactedTopicTest.test_upload.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#019645fc-4c13-4dfe-abf4-48e13517dc4b | FLAKY | 12/21 |
| rptest.tests.tiered_storage_model_test.TieredStorageTest.test_tiered_storage.cloud_storage_type_and_url_style=.CloudStorageType.ABS.2.virtual_host.test_case=.TS_Read==True.SegmentRolledByTimeout==True | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#019645fc-4c13-4dfe-abf4-48e13517dc4b | FAIL | 0/1 |
| rptest.tests.topic_delete_test.TopicDeleteCloudStorageTest.topic_delete_installed_snapshots_test | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#01964600-f35d-44e8-aad2-585383bf51a2 | FLAKY | 5/21 |
| rptest.tests.upgrade_test.UpgradeFromPriorFeatureVersionCloudStorageTest.test_rolling_upgrade.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#019645fc-4c14-464e-a515-a1099fff198b | FAIL | 0/21 |
| rptest.tests.upgrade_test.UpgradeFromPriorFeatureVersionCloudStorageTest.test_rolling_upgrade.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64707#01964600-f35b-4852-b807-fabe7a160990 | FAIL | 0/21 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64719">build#64719</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_aborted_tx.recovery_overrides=.retention.local.target.bytes.1024.redpanda.remote.write.True.redpanda.remote.read.True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64719#019646dc-398d-4cf1-be06-4bdcad0615de | FLAKY | 5/21 |
| rptest.tests.offset_for_leader_epoch_archival_test.OffsetForLeaderEpochArchivalTest.test_querying_archive | ducktape | https://buildkite.com/redpanda/redpanda/builds/64719#019646dc-3988-4faa-b616-7c07e2aef792 | FAIL | 0/21 |
| rptest.tests.offset_for_leader_epoch_archival_test.OffsetForLeaderEpochArchivalTest.test_querying_archive | ducktape | https://buildkite.com/redpanda/redpanda/builds/64719#019646dc-398e-484c-a1e4-e06372fdf863 | FAIL | 0/21 |
| rptest.tests.read_replica_e2e_test.ReadReplicasUpgradeTest.test_upgrades.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64719#019646dc-3985-46b7-bbca-8468c67985a1 | FAIL | 0/21 |
| rptest.tests.read_replica_e2e_test.ReadReplicasUpgradeTest.test_upgrades.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64719#019646dc-398b-4679-8dc7-701b434c72ce | FAIL | 0/21 |
| rptest.tests.shadow_indexing_compacted_topic_test.ShadowIndexingCompactedTopicTest.test_upload.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64719#019646dc-3986-4724-ab75-9a3eb0b947be | FLAKY | 10/21 |
| rptest.tests.upgrade_test.UpgradeFromPriorFeatureVersionCloudStorageTest.test_rolling_upgrade.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64719#019646dc-3987-4fd0-a8ba-a9575c0bb01a | FAIL | 0/21 |
| rptest.tests.upgrade_test.UpgradeFromPriorFeatureVersionCloudStorageTest.test_rolling_upgrade.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64719#019646dc-398e-484c-a1e4-e06372fdf863 | FAIL | 0/21 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64725">build#64725</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=False.params=.cancellation.dir.in.stage.preparing.use_alias.True | ducktape | https://buildkite.com/redpanda/redpanda/builds/64725#0196479c-5cb4-451a-a3a4-f1ccc29e80d9 | FLAKY | 20/21 |
| rptest.tests.datalake.throttling_test.DatalakeThrottlingTest.test_basic_throttling.cloud_storage_type=CloudStorageType.S3.catalog_type=CatalogType.NESSIE | ducktape | https://buildkite.com/redpanda/redpanda/builds/64725#0196479c-5cb6-4f3d-91fd-f2de333e88fb | FLAKY | 17/21 |
| rptest.tests.datalake.throttling_test.DatalakeThrottlingTest.test_basic_throttling.cloud_storage_type=CloudStorageType.S3.catalog_type=CatalogType.REST_JDBC | ducktape | https://buildkite.com/redpanda/redpanda/builds/64725#0196479c-5cb4-451a-a3a4-f1ccc29e80d9 | FLAKY | 17/21 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_aborted_tx.recovery_overrides=.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64725#0196479c-5cb6-4f3d-91fd-f2de333e88fb | FLAKY | 5/21 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_aborted_tx.recovery_overrides=.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64725#0196479c-5cb8-4e7a-b9cb-844e4c8aec08 | FLAKY | 9/21 |
| rptest.tests.e2e_topic_recovery_test.EndToEndTopicRecovery.test_restore_with_aborted_tx.recovery_overrides=.retention.local.target.bytes.1024.redpanda.remote.write.True.redpanda.remote.read.True.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64725#0196479c-5cb5-4755-ab72-51798e4b878a | FLAKY | 5/21 |
| rptest.tests.random_node_operations_test.RandomNodeOperationsTest.test_node_operations.enable_failures=True.mixed_versions=False.with_tiered_storage=False.with_iceberg=True.with_chunked_compaction=False.cloud_storage_type=CloudStorageType.S3 | ducktape | https://buildkite.com/redpanda/redpanda/builds/64725#0196479c-5cb5-4755-ab72-51798e4b878a | FLAKY | 20/21 |
| rptest.tests.upgrade_test.UpgradeBackToBackTest.test_upgrade_with_all_workloads.single_upgrade=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64725#0196479c-5cb6-4f3d-91fd-f2de333e88fb | FLAKY | 17/21 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64803">build#64803</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.consumer_group_balancing_test.ConsumerGroupBalancingTest.test_coordinator_nodes_balance | ducktape | https://buildkite.com/redpanda/redpanda/builds/64803#01965756-a0c6-4d0e-bccc-86e6e050ca27 | FLAKY | 20/21 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64803#0196573b-e3a6-48b6-a94f-e78482caf774 | FLAKY | 12/21 |
| rptest.tests.datalake.datalake_e2e_test.DatalakeE2ETests.test_topic_lifecycle.cloud_storage_type=CloudStorageType.S3.catalog_type=CatalogType.REST_HADOOP | ducktape | https://buildkite.com/redpanda/redpanda/builds/64803#0196573b-e3a5-4acb-be35-2fdeb811ad08 | FLAKY | 20/21 |
| rptest.tests.scaling_up_test.ScalingUpTest.test_scaling_up_with_recovered_topic | ducktape | https://buildkite.com/redpanda/redpanda/builds/64803#01965756-a0c5-4f8c-a593-3141b0285a4d | FLAKY | 12/21 |
| rptest.tests.shadow_indexing_compacted_topic_test.ShadowIndexingCompactedTopicTest.test_upload.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64803#0196573b-e3a6-48b6-a94f-e78482caf774 | FLAKY | 11/21 |
| rptest.tests.upgrade_test.UpgradeBackToBackTest.test_upgrade_with_all_workloads.single_upgrade=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64803#0196573b-e3a6-48b6-a94f-e78482caf774 | FLAKY | 14/21 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64808">build#64808</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64808#01965942-433e-4a70-ac57-4503eff700c0 | FLAKY | 18/25 |
| rptest.tests.scaling_up_test.ScalingUpTest.test_scaling_up_with_recovered_topic | ducktape | https://buildkite.com/redpanda/redpanda/builds/64808#01965942-4347-401e-9f9c-7ebc2f08aa06 | FLAKY | 18/25 |
| rptest.tests.shadow_indexing_compacted_topic_test.ShadowIndexingCompactedTopicTest.test_upload.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64808#01965942-4340-4299-b383-73c203b6f7ee | FLAKY | 16/25 |
| rptest.tests.upgrade_test.UpgradeBackToBackTest.test_upgrade_with_all_workloads.single_upgrade=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64808#01965942-433d-461a-aafc-5638a49f3a6e | FLAKY | 13/25 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64811">build#64811</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64811#019659f9-7931-448b-bdcb-23ae382c93bf | FLAKY | 35/45 |
| rptest.tests.scaling_up_test.ScalingUpTest.test_scaling_up_with_recovered_topic | ducktape | https://buildkite.com/redpanda/redpanda/builds/64811#019659f9-793a-479d-84f7-d2ff598dd723 | FLAKY | 30/45 |
| rptest.tests.shadow_indexing_compacted_topic_test.ShadowIndexingCompactedTopicTest.test_upload.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64811#019659f9-7933-4663-943d-cb86ac6b03a0 | FLAKY | 27/45 |
| rptest.tests.upgrade_test.UpgradeBackToBackTest.test_upgrade_with_all_workloads.single_upgrade=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64811#019659f9-7930-40f5-beda-f8d2140c341c | FLAKY | 30/45 |
| rptest.tests.upgrade_test.UpgradeBackToBackTest.test_upgrade_with_all_workloads.single_upgrade=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64811#019659f9-7939-461c-ac49-36b0b210a89b | FLAKY | 35/45 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64822">build#64822</a></summary>


| test_id | test_kind | job_url | test_status | passed |
|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| gtest_raft_rpunit.gtest_raft_rpunit | unit | https://buildkite.com/redpanda/redpanda/builds/64822#01965b90-422a-468a-a42d-f3013b15fb75 | FLAKY | 1/2 |
| rptest.tests.data_migrations_api_test.DataMigrationsApiTest.test_migrated_topic_data_integrity.transfer_leadership=True.params=.cancellation.dir.in.stage.preparing.use_alias.True | ducktape | https://buildkite.com/redpanda/redpanda/builds/64822#01965be7-d71d-4be1-8bb0-310498457ff3 | FLAKY | 17/21 |
| rptest.tests.scaling_up_test.ScalingUpTest.test_scaling_up_with_recovered_topic | ducktape | https://buildkite.com/redpanda/redpanda/builds/64822#01965bd5-d4aa-4351-8562-19204dd15603 | FLAKY | 18/21 |
| rptest.tests.shadow_indexing_compacted_topic_test.ShadowIndexingCompactedTopicTest.test_upload.cloud_storage_type=CloudStorageType.ABS | ducktape | https://buildkite.com/redpanda/redpanda/builds/64822#01965be7-d722-44f6-bea1-dabb95d4fe0d | FLAKY | 9/21 |
| rptest.tests.upgrade_test.UpgradeBackToBackTest.test_upgrade_with_all_workloads.single_upgrade=False | ducktape | https://buildkite.com/redpanda/redpanda/builds/64822#01965bd5-d4ab-4521-9e00-fbb76037a646 | FLAKY | 15/21 |
</details>
<details>
<summary>test results on <a href="https://buildkite.com/redpanda/redpanda/builds/64869">build#64869</a></summary>


| test_id                                                                                                                               | test_kind   | job_url                                                                                   | test_status   | passed   |
|:--------------------------------------------------------------------------------------------------------------------------------------|:------------|:------------------------------------------------------------------------------------------|:--------------|:---------|
| rptest.tests.shadow_indexing_compacted_topic_test.ShadowIndexingCompactedTopicTest.test_upload.cloud_storage_type=CloudStorageType.S3 | ducktape    | https://buildkite.com/redpanda/redpanda/builds/64869#01965ebb-4cd4-4c03-840b-87670e3d792e | FLAKY         | 11/21    |
</details>
<!--/body-->

oleiman committed:
fixes (will squash back)

- allow uploading compacted segments in new_non_compacted mode
  - TODO: rename mode 'new_upload' or whatever
- ignore segments where upload range doesn't include any data batches
- discard candidate if dirty offset exceeds LSO

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64707-->
<!--4129c4a5-4cc0-4f5a-8b30-6a2d541c8ed7-->
<!--67d1e2d8-3dbe-40fe-8519-1e0ce14febf3-->
<!--2a475a8e-5e75-4f26-9945-0214233a47d9-->
<!--3e4ffabe-2d2a-4891-a4aa-d8b90fee8fc1-->
<!--466f9da6-6c46-499b-84d2-155577070fce-->
<!--b81cf095-1169-4ded-828b-b37901673a2a-->
<!--bd575a91-0980-4137-b0a4-d892121aef1a-->
<!--title-->
## Retry command for [Build#64707](https://buildkite.com/redpanda/redpanda/builds/64707)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```


/ci-repeat 1
tests/rptest/tests/upgrade_test.py::UpgradeFromPriorFeatureVersionCloudStorageTest.test_rolling_upgrade@{"cloud_storage_type":1}
tests/rptest/tests/read_replica_e2e_test.py::ReadReplicasUpgradeTest.test_upgrades@{"cloud_storage_type":1}
tests/rptest/tests/data_migrations_api_test.py::DataMigrationsApiTest.test_migrated_topic_data_integrity@{"params":{"cancellation":{"dir":"in","stage":"preparing"},"use_alias":true},"transfer_leadership":true}
tests/rptest/tests/offset_for_leader_epoch_archival_test.py::OffsetForLeaderEpochArchivalTest.test_querying_archive
tests/rptest/tests/shadow_indexing_compacted_topic_test.py::ShadowIndexingCompactedTopicTest.test_upload@{"cloud_storage_type":1}
tests/rptest/tests/tiered_storage_model_test.py::TieredStorageTest.test_tiered_storage@{"cloud_storage_type_and_url_style":[2,"virtual_host"],"test_case":{"name":"(TS_Read == True, SegmentRolledByTimeout == True)"}}
tests/rptest/tests/datalake/compaction_test.py::CompactionTest.test_compaction@{"catalog_type":"rest_hadoop","cloud_storage_type":1}
tests/rptest/tests/cloud_storage_timing_stress_test.py::CloudStorageTimingStressTest.test_cloud_storage_with_partition_moves@{"cleanup_policy":"delete"}
tests/rptest/tests/e2e_topic_recovery_test.py::EndToEndTopicRecovery.test_restore_with_aborted_tx@{"cloud_storage_type":1,"recovery_overrides":{"redpanda.remote.read":true,"redpanda.remote.write":true,"retention.local.target.bytes":1024}}
tests/rptest/tests/datalake/throttling_test.py::DatalakeThrottlingTest.test_basic_throttling@{"catalog_type":"nessie","cloud_storage_type":1}
tests/rptest/tests/topic_delete_test.py::TopicDeleteCloudStorageTest.topic_delete_installed_snapshots_test
tests/rptest/tests/datalake/compaction_test.py::CompactionTest.test_compaction@{"catalog_type":"nessie","cloud_storage_type":1}
tests/rptest/tests/data_migrations_api_test.py::DataMigrationsApiTest.test_higher_level_migration_api
```
<!--/body-->

oleiman commented:
/ci-repeat 1
skip-redpanda-build
skip-units
tests/rptest/tests/upgrade_test.py::UpgradeFromPriorFeatureVersionCloudStorageTest.test_rolling_upgrade@{"cloud_storage_type":1}
tests/rptest/tests/read_replica_e2e_test.py::ReadReplicasUpgradeTest.test_upgrades@{"cloud_storage_type":1}
tests/rptest/tests/data_migrations_api_test.py::DataMigrationsApiTest.test_migrated_topic_data_integrity@{"params":{"cancellation":{"dir":"in","stage":"preparing"},"use_alias":true},"transfer_leadership":true}
tests/rptest/tests/offset_for_leader_epoch_archival_test.py::OffsetForLeaderEpochArchivalTest.test_querying_archive
tests/rptest/tests/shadow_indexing_compacted_topic_test.py::ShadowIndexingCompactedTopicTest.test_upload@{"cloud_storage_type":1}
tests/rptest/tests/tiered_storage_model_test.py::TieredStorageTest.test_tiered_storage@{"cloud_storage_type_and_url_style":[2,"virtual_host"],"test_case":{"name":"(TS_Read == True, SegmentRolledByTimeout == True)"}}
tests/rptest/tests/datalake/compaction_test.py::CompactionTest.test_compaction@{"catalog_type":"rest_hadoop","cloud_storage_type":1}
tests/rptest/tests/cloud_storage_timing_stress_test.py::CloudStorageTimingStressTest.test_cloud_storage_with_partition_moves@{"cleanup_policy":"delete"}
tests/rptest/tests/e2e_topic_recovery_test.py::EndToEndTopicRecovery.test_restore_with_aborted_tx@{"cloud_storage_type":1,"recovery_overrides":{"redpanda.remote.read":true,"redpanda.remote.write":true,"retention.local.target.bytes":1024}}
tests/rptest/tests/datalake/throttling_test.py::DatalakeThrottlingTest.test_basic_throttling@{"catalog_type":"nessie","cloud_storage_type":1}
tests/rptest/tests/topic_delete_test.py::TopicDeleteCloudStorageTest.topic_delete_installed_snapshots_test
tests/rptest/tests/datalake/compaction_test.py::CompactionTest.test_compaction@{"catalog_type":"nessie","cloud_storage_type":1}
tests/rptest/tests/data_migrations_api_test.py::DataMigrationsApiTest.test_higher_level_migration_api

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64719-->
<!--12075812-1354-47b5-8c50-eb3e1ff1d706-->
<!--0cef2dec-eb62-4896-847d-8b962bdda9a9-->
<!--868eeca6-9b97-4666-be70-fb2ac557a585-->
<!--4a3af940-0d98-4b63-98a0-e4eb3b2319b7-->
<!--edf225e4-adb0-4757-840e-ce43aabbd4bd-->
<!--8f2cbba8-f850-4187-a3c6-f029f3bee5b2-->
<!--dd62f2b9-4499-4d8e-afa6-04d2466797b3-->
<!--title-->
## Retry command for [Build#64719](https://buildkite.com/redpanda/redpanda/builds/64719)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```


/ci-repeat 1
tests/rptest/tests/upgrade_test.py::UpgradeFromPriorFeatureVersionCloudStorageTest.test_rolling_upgrade@{"cloud_storage_type":1}
tests/rptest/tests/read_replica_e2e_test.py::ReadReplicasUpgradeTest.test_upgrades@{"cloud_storage_type":1}
tests/rptest/tests/offset_for_leader_epoch_archival_test.py::OffsetForLeaderEpochArchivalTest.test_querying_archive
tests/rptest/tests/e2e_topic_recovery_test.py::EndToEndTopicRecovery.test_restore_with_aborted_tx@{"cloud_storage_type":1,"recovery_overrides":{"redpanda.remote.read":true,"redpanda.remote.write":true,"retention.local.target.bytes":1024}}
tests/rptest/tests/shadow_indexing_compacted_topic_test.py::ShadowIndexingCompactedTopicTest.test_upload@{"cloud_storage_type":1}
```
<!--/body-->

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64725-->
<!--2b13ded8-07f3-41ab-8771-a66d4c50a67b-->
<!--0159e343-494d-4893-aa2d-90584b9f470d-->
<!--06362ed7-179e-4a89-98e4-64689d1136cb-->
<!--62104ede-a699-4298-8c1d-547ce52ca726-->
<!--title-->
## Retry command for [Build#64725](https://buildkite.com/redpanda/redpanda/builds/64725)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```


/ci-repeat 1
tests/rptest/tests/data_migrations_api_test.py::DataMigrationsApiTest.test_migrated_topic_data_integrity@{"params":{"cancellation":{"dir":"in","stage":"preparing"},"use_alias":true},"transfer_leadership":false}
tests/rptest/tests/datalake/throttling_test.py::DatalakeThrottlingTest.test_basic_throttling@{"catalog_type":"rest_jdbc","cloud_storage_type":1}
tests/rptest/tests/e2e_topic_recovery_test.py::EndToEndTopicRecovery.test_restore_with_aborted_tx@{"cloud_storage_type":1,"recovery_overrides":{}}
tests/rptest/tests/e2e_topic_recovery_test.py::EndToEndTopicRecovery.test_restore_with_aborted_tx@{"cloud_storage_type":2,"recovery_overrides":{}}
tests/rptest/tests/datalake/throttling_test.py::DatalakeThrottlingTest.test_basic_throttling@{"catalog_type":"nessie","cloud_storage_type":1}
tests/rptest/tests/random_node_operations_test.py::RandomNodeOperationsTest.test_node_operations@{"cloud_storage_type":1,"enable_failures":true,"mixed_versions":false,"with_chunked_compaction":false,"with_iceberg":true,"with_tiered_storage":false}
tests/rptest/tests/e2e_topic_recovery_test.py::EndToEndTopicRecovery.test_restore_with_aborted_tx@{"cloud_storage_type":1,"recovery_overrides":{"redpanda.remote.read":true,"redpanda.remote.write":true,"retention.local.target.bytes":1024}}
```
<!--/body-->

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64797-->
<!--fb819fa4-83d8-44ad-9333-407e695eba03-->
<!--6bb52fad-60cb-4767-ab35-62117225c636-->
<!--3199482a-fb67-4866-8a1e-c276cfcfd373-->
<!--89e071b0-ac41-4206-99d6-7eafa5613b73-->
<!--title-->
## Retry command for [Build#64797](https://buildkite.com/redpanda/redpanda/builds/64797)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```


/ci-repeat 1
tests/rptest/tests/read_replica_e2e_test.py::TestReadReplicaService.test_simple_end_to_end@{"cloud_storage_type_and_url_style":[1,"path"],"partition_count":10}
tests/rptest/tests/availability_test.py::AvailabilityTests.test_availability_when_one_node_failed
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_fast2@{"cloud_storage_type":1}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_many_partitions@{"check_mode":"check_manifest_existence","cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_many_partitions@{"check_mode":"no_check","cloud_storage_type":1}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_size_based_retention@{"cloud_storage_type":1}
tests/rptest/tests/read_replica_e2e_test.py::TestReadReplicaService.test_simple_end_to_end@{"cloud_storage_type_and_url_style":[1,"virtual_host"],"partition_count":10}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_admin_api_recovery@{"cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_fast1@{"cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_fast3@{"cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_prevent_recovery@{"cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_many_partitions@{"check_mode":"no_check","cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_time_based_retention@{"cloud_storage_type":2}
tests/rptest/tests/read_replica_e2e_test.py::TestReadReplicaService.test_simple_end_to_end@{"cloud_storage_type_and_url_style":[2,"virtual_host"],"partition_count":10}
tests/rptest/tests/cloud_storage_scrubber_test.py::CloudStorageScrubberTest.test_scrubber@{"cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_fast2@{"cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_many_partitions@{"check_mode":"check_manifest_and_segment_metadata","cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_many_partitions@{"check_mode":"check_manifest_existence","cloud_storage_type":1}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_size_based_retention@{"cloud_storage_type":2}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_admin_api_recovery@{"cloud_storage_type":1}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_fast1@{"cloud_storage_type":1}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_fast3@{"cloud_storage_type":1}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_prevent_recovery@{"cloud_storage_type":1}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_time_based_retention@{"cloud_storage_type":1}
tests/rptest/tests/topic_recovery_test.py::TopicRecoveryTest.test_many_partitions@{"check_mode":"check_manifest_and_segment_metadata","cloud_storage_type":1}
```
<!--/body-->

oleiman commented:
> https://buildkite.com/redpanda/redpanda/builds/64797

whoops. introduced a careless logic error right before pushing.

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64803-->
<!--f18f9205-3bcc-417e-97ab-08e95b5e381e-->
<!--a62548e9-a78f-4962-9d4b-2b5abcb770ac-->
<!--278f785a-6b74-49d3-8425-5edfd70f11a2-->
<!--112b3e7d-f8a7-4d78-89be-e9508bff0330-->
<!--title-->
## Retry command for [Build#64803](https://buildkite.com/redpanda/redpanda/builds/64803)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```


/ci-repeat 1
tests/rptest/tests/datalake/datalake_e2e_test.py::DatalakeE2ETests.test_topic_lifecycle@{"catalog_type":"rest_hadoop","cloud_storage_type":1}
tests/rptest/tests/data_migrations_api_test.py::DataMigrationsApiTest.test_migrated_topic_data_integrity@{"params":{"cancellation":{"dir":"in","stage":"preparing"},"use_alias":false},"transfer_leadership":true}
tests/rptest/tests/shadow_indexing_compacted_topic_test.py::ShadowIndexingCompactedTopicTest.test_upload@{"cloud_storage_type":2}
tests/rptest/tests/upgrade_test.py::UpgradeBackToBackTest.test_upgrade_with_all_workloads@{"single_upgrade":false}
tests/rptest/tests/scaling_up_test.py::ScalingUpTest.test_scaling_up_with_recovered_topic
tests/rptest/tests/consumer_group_balancing_test.py::ConsumerGroupBalancingTest.test_coordinator_nodes_balance
```
<!--/body-->

oleiman commented:
/ci-repeat 1
skip-redpanda-build
skip-units
tests/rptest/tests/datalake/datalake_e2e_test.py::DatalakeE2ETests.test_topic_lifecycle@{"catalog_type":"rest_hadoop","cloud_storage_type":1}
tests/rptest/tests/data_migrations_api_test.py::DataMigrationsApiTest.test_migrated_topic_data_integrity@{"params":{"cancellation":{"dir":"in","stage":"preparing"},"use_alias":false},"transfer_leadership":true}
tests/rptest/tests/shadow_indexing_compacted_topic_test.py::ShadowIndexingCompactedTopicTest.test_upload@{"cloud_storage_type":2}
tests/rptest/tests/upgrade_test.py::UpgradeBackToBackTest.test_upgrade_with_all_workloads@{"single_upgrade":false}
tests/rptest/tests/scaling_up_test.py::ScalingUpTest.test_scaling_up_with_recovered_topic
tests/rptest/tests/consumer_group_balancing_test.py::ConsumerGroupBalancingTest.test_coordinator_nodes_balance
dt-repeat=5

oleiman commented:
/ci-repeat 1
skip-redpanda-build
skip-units
dt-repeat=5
tests/rptest/tests/datalake/datalake_e2e_test.py::DatalakeE2ETests.test_topic_lifecycle@{"catalog_type":"rest_hadoop","cloud_storage_type":1}
tests/rptest/tests/data_migrations_api_test.py::DataMigrationsApiTest.test_migrated_topic_data_integrity@{"params":{"cancellation":{"dir":"in","stage":"preparing"},"use_alias":false},"transfer_leadership":true}
tests/rptest/tests/shadow_indexing_compacted_topic_test.py::ShadowIndexingCompactedTopicTest.test_upload@{"cloud_storage_type":2}
tests/rptest/tests/upgrade_test.py::UpgradeBackToBackTest.test_upgrade_with_all_workloads@{"single_upgrade":false}
tests/rptest/tests/scaling_up_test.py::ScalingUpTest.test_scaling_up_with_recovered_topic
tests/rptest/tests/consumer_group_balancing_test.py::ConsumerGroupBalancingTest.test_coordinator_nodes_balance

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64808-->
<!--76009f49-208a-4aa1-87d6-d80317a04de0-->
<!--7dd98a42-7619-4f12-a5d0-3a47796afaca-->
<!--title-->
## Retry command for [Build#64808](https://buildkite.com/redpanda/redpanda/builds/64808)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```

/ci-repeat 1
tests/rptest/tests/data_migrations_api_test.py::DataMigrationsApiTest.test_migrated_topic_data_integrity@{"params":{"cancellation":{"dir":"in","stage":"preparing"},"use_alias":false},"transfer_leadership":true}
tests/rptest/tests/upgrade_test.py::UpgradeBackToBackTest.test_upgrade_with_all_workloads@{"single_upgrade":false}
```
<!--/body-->

oleiman commented:
CI Failures ([build](https://buildkite.com/redpanda/redpanda/builds/64808)):
- One looks like https://redpandadata.atlassian.net/browse/CORE-9506 (DataMigrationsApiTest flake)
- Other one looks like https://redpandadata.atlassian.net/browse/CORE-8311 (UpgradeBackToBackTest)

CI Failures ([build](https://buildkite.com/redpanda/redpanda/builds/64803)):
- https://redpandadata.atlassian.net/browse/CORE-8764 (scrubber detected anomalies in ScalingUpTest)
-

oleiman commented:
/ci-repeat 1
skip-redpanda-build
skip-units
dt-repeat=25
tests/rptest/tests/datalake/datalake_e2e_test.py::DatalakeE2ETests.test_topic_lifecycle@{"catalog_type":"rest_hadoop","cloud_storage_type":1}
tests/rptest/tests/data_migrations_api_test.py::DataMigrationsApiTest.test_migrated_topic_data_integrity@{"params":{"cancellation":{"dir":"in","stage":"preparing"},"use_alias":false},"transfer_leadership":true}
tests/rptest/tests/shadow_indexing_compacted_topic_test.py::ShadowIndexingCompactedTopicTest.test_upload@{"cloud_storage_type":2}
tests/rptest/tests/upgrade_test.py::UpgradeBackToBackTest.test_upgrade_with_all_workloads@{"single_upgrade":false}
tests/rptest/tests/scaling_up_test.py::ScalingUpTest.test_scaling_up_with_recovered_topic
tests/rptest/tests/consumer_group_balancing_test.py::ConsumerGroupBalancingTest.test_coordinator_nodes_balance

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64811-->
<!--cd487195-ab72-4f63-bee9-468382053d83-->
<!--7caf9843-a709-4697-9abc-353b7b42155d-->
<!--d829c05b-e139-44e1-b190-8c6d26736223-->
<!--title-->
## Retry command for [Build#64811](https://buildkite.com/redpanda/redpanda/builds/64811)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```


/ci-repeat 1
tests/rptest/tests/scaling_up_test.py::ScalingUpTest.test_scaling_up_with_recovered_topic
tests/rptest/tests/upgrade_test.py::UpgradeBackToBackTest.test_upgrade_with_all_workloads@{"single_upgrade":false}
```
<!--/body-->

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64822-->
<!--0d019e67-258c-4e20-b637-caed74b0db84-->
<!--b398dbba-e2ea-44bf-b3f8-65ae3e97b027-->
<!--title-->
## Retry command for [Build#64822](https://buildkite.com/redpanda/redpanda/builds/64822)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```

/ci-repeat 1
tests/rptest/tests/data_migrations_api_test.py::DataMigrationsApiTest.test_migrated_topic_data_integrity@{"params":{"cancellation":{"dir":"in","stage":"preparing"},"use_alias":true},"transfer_leadership":true}
tests/rptest/tests/shadow_indexing_compacted_topic_test.py::ShadowIndexingCompactedTopicTest.test_upload@{"cloud_storage_type":2}
```
<!--/body-->

vbotbuildovich commented:
<!--FAILURES SLASH COMMAND https://buildkite.com/redpanda/redpanda/builds/64869-->
<!--eecfc56f-53fe-49a7-863e-4d0e97dd5a64-->
<!--title-->
## Retry command for [Build#64869](https://buildkite.com/redpanda/redpanda/builds/64869)
please wait until all jobs are finished before running the slash command
<!--/title-->
<!--body-->
```
/ci-repeat 1
tests/rptest/tests/shadow_indexing_compacted_topic_test.py::ShadowIndexingCompactedTopicTest.test_upload@{"cloud_storage_type":1}
```
<!--/body-->
